pyspark-notebook/Dockerfile

```diff
@@ -13,7 +13,8 @@ ENV HADOOP_VERSION 2.7
 RUN apt-get -y update && \
     apt-get install --no-install-recommends -y openjdk-8-jre-headless ca-certificates-java && \
     apt-get clean && \
-    rm -rf /var/lib/apt/lists/*
+    rm -rf /var/lib/apt/lists/* && \
+    conda install -y pyarrow
```

Comment:
I think it's reasonable to install pyarrow in the pyspark image. If there's already a section that installs packages using `conda`, please add `pyarrow` there. If there, isn't it's best to start one since we need to run some cleanup and a script to fix permissions as part of the same line. https://github.com/jupyter/docker-stacks/blob/master/scipy-notebook/Dockerfile#L20 can serve as a guide.

Reply:
@parente Good point, I've updated the PR.